src/libs/export.c

```diff
@@ -192,11 +202,15 @@ static void export_button_clicked(GtkWidget *widget, dt_lib_export_t *d)
   gchar *icc_filename = dt_conf_get_string(CONFIG_PREFIX "iccprofile");
   dt_iop_color_intent_t icc_intent = dt_conf_get_int(CONFIG_PREFIX "iccintent");
 
+  gtk_entry_set_text (GTK_ENTRY(d->scale), dt_conf_get_string("plugins/lighttable/export/resizing_factor"));
+
   GList *list = g_list_copy((GList *)dt_view_get_images_to_act_on(TRUE, TRUE));
   dt_control_export(list, max_width, max_height, format_index, storage_index, high_quality, upscale, export_masks,
                     style, style_append, icc_type, icc_filename, icc_intent, d->metadata_export);
 
   g_free(icc_filename);
+
+  dt_imageio_scale_optim();
```

Comment:
Can't we have the cale optim routine internal to export.c ? I don't like having that exported as routine of imageio.

Reply:
Yes, it has become possible. Did.